webpack/scenes/RedHatRepositories/index.scss

```diff
@@ -24,4 +24,48 @@
       white-space: pre-wrap;
     }
   }
+
+  .list-view-pf-view {
+    background-color: inherit;
+  }
+
+  .available-repositories-container {
+    background-color: $color-pf-white;
+    border-right: $color-pf-black-300 2px solid;
+  }
+
+  .content-view-pf-pagination {
+    background-color: $color-pf-white;
+    margin-bottom: 20px;
+  }
+
+  .toolbar-pf-filter {
+    margin-bottom: 7px;
+  }
+
+  @media (min-width: 1280px) {
+    .toolbar-pf-filter {
+      width: 50%;
+
+      .input-group {
+        width: 400px;
+      }
+    }
+  }
+
+  .enabled-repositories-container {
+    .pficon.list-view-pf-icon-sm {
+      border-color: $color-pf-green-300;
+    }
+  }
+
+  //TODO: for some reason these are missing in our patternfly version
```

Comment:
I don't know for sure but I think it's likely related to http://projects.theforeman.org/issues/21139 since we are still using an out of date patternfly-sass version